CHANGELOG.md

```diff
@@ -9,14 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added support for the HuggingFace Hub as an alternative way to handle loading files.
```

Comment:
If we go with `huggingface://` or `hf://` type urls, we should mention that here.

Reply:
I went with `hf://`! I updated the changelog.